sympy/solvers/tests/test_ode.py

```diff
@@ -1115,21 +1118,21 @@ def test_solve_ics():
 
     # Some more complicated tests Refer to PR #16098
 
-    assert dsolve(f(x).diff(x)*(f(x).diff(x, 2)-x), ics={f(0):0, f(x).diff(x).subs(x, 1):0}) == \
-        [Eq(f(x), 0), Eq(f(x), x ** 3 / 6 - x / 2)]
-    assert dsolve(f(x).diff(x)*(f(x).diff(x, 2)-x), ics={f(0):0}) == \
-        [Eq(f(x), 0), Eq(f(x), C2*x + x**3/6)]
+    assert set(dsolve(f(x).diff(x)*(f(x).diff(x, 2)-x), ics={f(0):0, f(x).diff(x).subs(x, 1):0})) == \
+        set([Eq(f(x), 0), Eq(f(x), x ** 3 / 6 - x / 2)])
+    assert set(dsolve(f(x).diff(x)*(f(x).diff(x, 2)-x), ics={f(0):0})) == \
+        set([Eq(f(x), 0), Eq(f(x), C2*x + x**3/6)])
```

Comment:
You can create a set using `{}`

Reply:
okay

Reply:
Not with list
{list} will throw an error.

Reply:
Instead of writing `set([a, b])` you can write `{a, b}`.